spec/reek/examiner_spec.rb

```diff
@@ -108,5 +108,42 @@
         expect(examiner.smells).to eq([])
       end
     end
+
+    context 'with an incomprehensible source that causes Reek to crash' do
+      let(:subject) do
+        examiner = described_class.new 'class C; def does_crash_reek; end; end'
+        allow(examiner).to receive(:smell_repository) do
```

Comment:
I'm don't like it if the object under test is stubbed. Can we avoid this by either injecting `SmellRepository`, or stubbing `SmellRepository.new` directly?